### ISSUE TITLE
Alpha version feedback

### ISSUE THREAD

**Issue by frenzymadness** [frenzymadness/pyDobble#2]
- [ ] Add some margin to each card
- [ ] Use better paper so logos won't be visible from other side
- [x] Remove text from cookiecutter logo
- [ ] Rotate logos more than before

**Comment by torsava** [comment]
Some of the logos were really close together as well, so maybe a bit more space in between.

**Comment by frenzymadness** [comment]
@torsava Next version will have one more logo on each card, so will see what will be possible.

**Comment by torsava** [comment]
Yeah, the logos on the original game were a lot more simple. Maybe we could make the playing cards a tiny bit bigger since on the big printer we won't be limited to the A4 paper.

We might be able to print an A3 in the office as well, I haven't tried it yet.

**Comment by frenzymadness** [comment]
@hroncok As we discussed yesterday, could you please try to find some time to make a beta version of cards? Lists can be simply generated with `./generator.py` command.

**Comment by hroncok** [comment]
I will give it a try.

**Comment by hroncok** [comment]
This have slipped from my radar. will try again.

**Comment by frenzymadness** [comment]
bump :smile:

**Comment by hroncok** [comment]
I have it on my radar, but other stuff (you know: work or the other work) somehow keeps occupying me.

**Comment by frenzymadness** [comment]
ping :slightly_smiling_face:

**Comment by hroncok** [comment]
![cards1 svg](https://user-images.githubusercontent.com/2401856/33216680-2e554d1c-d135-11e7-8cd2-81e0ebb50ff4.png)

Shall I continue in this direction?

**Comment by frenzymadness** [comment]
Awesome! I like it. Thank you!

**Comment by hroncok** [comment]
First sheet:

[cards1.pdf](https://github.com/frenzymadness/pycture-matching-game/files/1502490/cards1.pdf)

**Comment by frenzymadness** [comment]
I think that it's perfect. Could you please prepare other sheets? It'd be nice to have the game ready for Devconf. @hroncok @torsava Is it a reasonable deadline?

**Comment by hroncok** [comment]
I'll do my best.

**Comment by frenzymadness** [comment]
So, it wasn't a reasonable deadline :) Could you please take a look at this?

**Comment by frenzymadness** [comment]
Bump!

**Comment by hroncok** [comment]
Keep bumping. I need it. I've managed to promis too much things.

**Comment by frenzymadness** [comment]
Bump!

**Comment by hroncok** [comment]
Waiting for final pycon cz logo.

**Comment by edouardlib** [comment]
Hi @frenzymadness, great project! I'm willing to use it to help a teacher make some Dobbles games for education. 

make-svg.python doesn't seems to work as one function import doesn't exist (generate_matrix) do you know why is it missing?

**Comment by hroncok** [comment]
make-svg is old and does not work with new generator.py I guess

**Comment by frenzymadness** [comment]
Hello @edouardlib . Sorry for the later reply.

I fixed `make-svg.py` to work with generated JSON output from the generator and I also added more placeholders to the template so you can now generate SVGs with eight symbols on each card.

Take a look to readme and do not hesitate to ask for any help.

**Comment by edouardlib** [comment]
Thanks you for the update, look like it's working better.

I still got an error:
`  File "make-svg.py", line 63, in <module>
    generate_svgs(cards)
  File "make-svg.py", line 16, in generate_svgs
    lists = [template for _ in range(ceil(len(cards) / 12))]
TypeError: range() integer end argument expected, got float.`

I use personal set of images and then list them in the "list_of_project.txt".
Is there a typical amount of images to include? I personaly use a set of 18 images (svg).

I try to fix the code and run it but got a empty .svg result without contents. 

I am doing something wrong?

**Comment by frenzymadness** [comment]
If this comment doesn't solve your issue, you should open a new one because this is not related to the original topic.

It seems to me that you are using python 2. Am I right? If so, try to use Python 3 instead.

How big your set of images should depend on how many cards you want to create. You can generate sets with 7, 13, 31 or 57 cards and for each set of card, you need the same amount of symbols/images. If you have only 18 images, use `generator.py -o 3`.

Note that `make-svg.py` doesn't generate the final cards with images. It just fills the names of images to `templates/with-card-placeholders.svg` template so then you'll know where to put your images for each card.

**Comment by frenzymadness** [comment]
Ping @hroncok

**Comment by hroncok** [comment]
Ping me once the logo is finalized if you want this with 2019 logo.

**Comment by frenzymadness** [comment]
Ping @hroncok
